sim/encounters/blackwing_lair.go

```diff
@@ -122,182 +107,6 @@ func (ai *VaelastraszTheCorruptAI) registerSpells() {
-			spell.Dot(target).Apply(sim)
-			spell.Dot(target).AddStack(sim)
-		},
-	})
-
-	ai.cleaveSpell = ai.Target.RegisterSpell(core.SpellConfig{
-		ActionID:         cleaveActionID,
-		SpellSchool:      core.SpellSchoolPhysical,
-		DefenseType:      core.DefenseTypeMelee,
-		ProcMask:         core.ProcMaskEmpty,
-		DamageMultiplier: 1,
-
-		Cast: core.CastConfig{
-			CD: core.Cooldown{
-				Timer:    ai.Target.NewTimer(),
-				Duration: time.Second * 8,
-			},
-		},
-		ApplyEffects: func(sim *core.Simulation, target *core.Unit, spell *core.Spell) {
-			baseDamage := sim.Roll(5000, 8000)
-			spell.CalcAndDealDamage(sim, target, baseDamage, spell.OutcomeEnemyMeleeWhite)
-		},
-	})
-
-	ai.fireNovaSpell = ai.Target.RegisterSpell(core.SpellConfig{
-		ActionID:         fireNovaActionID,
-		SpellSchool:      core.SpellSchoolFire,
-		DefenseType:      core.DefenseTypeMagic,
-		ProcMask:         core.ProcMaskSpellDamage,
-		DamageMultiplier: 1,
-
-		Cast: core.CastConfig{
-			CD: core.Cooldown{
-				Timer:    ai.Target.NewTimer(),
-				Duration: time.Second * 3,
-			},
-		},
-		ApplyEffects: func(sim *core.Simulation, target *core.Unit, spell *core.Spell) {
-			baseDamage := sim.Roll(555.0, 645.0)
-			spell.CalcAndDealDamage(sim, target, baseDamage, spell.OutcomeMagicHit)
-		},
-	})
-
-	flameBreathTickDamage := 0.0
-
-	ai.flameBreathSpell = ai.Target.RegisterSpell(core.SpellConfig{
-		ActionID:         flameBreathActionID,
-		SpellSchool:      core.SpellSchoolFire,
-		DefenseType:      core.DefenseTypeMagic,
-		ProcMask:         core.ProcMaskSpellDamage,
-		DamageMultiplier: 1,
-
-		Cast: core.CastConfig{
-			DefaultCast: core.Cast{
-				GCD:      time.Millisecond * 3240, // Next server tick after cast complete
-				CastTime: time.Millisecond * 2000,
-			},
-			CD: core.Cooldown{
-				Timer:    ai.Target.NewTimer(),
-				Duration: time.Second * 9,
-			},
-			ModifyCast: func(sim *core.Simulation, spell *core.Spell, cast *core.Cast) {
-				spell.Unit.AutoAttacks.StopMeleeUntil(sim, sim.CurrentTime+cast.CastTime, false)
-			},
-		},
-		Dot: core.DotConfig{
-			Aura: core.Aura{
-				Label:     "Flame Breath",
-				MaxStacks: 100,
-				ActionID:  flameBreathActionID,
-				Duration:  time.Second * 15,
-				OnReset: func(aura *core.Aura, sim *core.Simulation) {
-					flameBreathTickDamage = 0.0
-				},
-			},
-			NumberOfTicks: 5,
-			TickLength:    time.Second * 3,
-			OnTick: func(sim *core.Simulation, target *core.Unit, dot *core.Dot) {
-				dot.Spell.CalcAndDealPeriodicDamage(sim, target, flameBreathTickDamage, dot.OutcomeTick)
-			},
-		},
-
-		ApplyEffects: func(sim *core.Simulation, target *core.Unit, spell *core.Spell) {
-			baseDamage := sim.Roll(3500, 4500)
-			result := spell.CalcDamage(sim, target, baseDamage, spell.OutcomeMagicHit)
-
-			if result.Landed() {
-				flameBreathTickDamage += sim.Roll(938, 1062)
-				spell.Dot(target).Activate(sim)
-				spell.Dot(target).AddStack(sim)
-			}
-			spell.DealDamage(sim, result)
-		},
-	})
-
```

Comment:
Can we leave all of these but commented out for now?

Reply:
Added